### ISSUE TITLE
Menu items

### ISSUE THREAD

**Issue by Seykotron** [oscarmlage/django-cruds-adminlte#140]
Hi guys,

While doing a very big project with this awesome framework we see that manage the menu in the _sidebar.html its not very good.

We have developed a model, a widget and a template tags wichs adds items to the menu, ordered and with perms.

If you like the feature I can do a PR.

Thank youuu!

**Comment by Kkeller83** [comment]
Sounds AMAZING!!! Would be lovely if you could share and/or create a PR.

**Comment by Seykotron** [comment]
Yeah, let oscarmlage to check if he likes the feature and I ask the PR.

We have it already developed and works very good, you can also include a template in the order you want if the features of perms are not enought for you.

Let Oscar reply and I ask the PR.

Thanks ;)

**Comment by solvo** [comment]
I implement something that could be interesting 
https://github.com/luisza/django-gentelella-widgets/blob/master/djgentelella/templatetags/gentelellamenu.py
This Menu management works with django models and can check perms, allow make custom menu widgets.
